ext/mysqlnd/config.w32

```diff
@@ -35,9 +35,9 @@ if (PHP_MYSQLND != "no") {
 			)
 		{
 			AC_DEFINE("MYSQLND_COMPRESSION_ENABLED", 1, "Define to 1 if mysqlnd has compressed protocol support.");
-			AC_DEFINE("MYSQLND_SSL_SUPPORTED", 1, "SSL support");
+			AC_DEFINE("MYSQLND_SSL_SUPPORTED", 1, "Define to 1 if mysqlnd core SSL is enabled.");
 			if (CHECK_LIB("crypt32.lib", "mysqlnd")) {
-				AC_DEFINE("MYSQLND_HAVE_SSL", 1, "Define to 1 if mysqlnd has extended SSL support through OpenSSL.");
+				AC_DEFINE("MYSQLND_HAVE_SSL", 1, "Define to 1 if mysqlnd extended SSL is enabled through OpenSSL.");
```

Comment:
This is not correct. While I'm not sure what's the difference between `MYSQLND_SSL_SUPPORTED` and `MYSQLND_HAVE_SSL` actually is, crypt32.lib has nothing to do with OpenSSL, but is a Windows library. That part has been added to support sha256 authentication even without OpenSSL.

Reply:
Resolved here: https://github.com/php/php-src/pull/16474